VocaDbWeb/Content/Styles/ExtLinks.less

```diff
@@ -300,3 +300,7 @@ a.extLink[href^="https://youtu.be/"] {
 a.extLink[href^="http://vocaloid.wikia.com/wiki/"] {
 	background: url("/Content/ExtIcons/vocawiki.png") no-repeat left 50%;
 }
+
+a.extLink[href^="https://www.vocaloid.com/"] {

```

Comment:
It's better to add the links using the format in the beginning of the file, like
```
&[href^="https://www.vocaloid.com/"] {
		.extLink(url("/Content/ExtIcons/vocaloidshop.png"));
	}
```

Reply:
Okay, thanks! I changed it. How is it now?